README.md

```diff
@@ -41,22 +41,48 @@ Checkout the example application at https://github.com/Moesif/moesif-elixir-phoe
    ]
    ```
 
-2. Configure the Plug in `config/runtime.exs`:
+2. Configure the Plug in as the example below. Replace `Your Moesif Application Id` with your Moesif application ID.  The others are optional with production defaults.
 
    ```elixir
    config :moesif_api, :config,
-     application_id: System.get_env("MOESIF_APPLICATION_ID") || "Your Moesif Application ID",
-     event_queue_size: String.to_integer(System.get_env("MOESIF_EVENT_QUEUE_SIZE") || "100000"),
-     max_batch_size: String.to_integer(System.get_env("MOESIF_MAX_BATCH_SIZE") || "10"),
-     max_batch_wait_time_ms: String.to_integer(System.get_env("MOESIF_MAX_BATCH_WAIT_TIME_MS") || "2000")
+      application_id: "Your Moesif Application Id",
+      event_queue_size: 100_000,
```

Comment:
@bakennedy is this format correct?

Reply:
@praves77 yes, the previous documented example config was just overengineering. Phoenix uses code for it's config files, and the example is just clearer using constants instead of pre-supposing env var fetching with defaults for the configuration.  Our default values are guaranteed to be applied by our library code rather than leaving it to the user to add as in the previous example, so it was just cluttering the readme previously